.github/workflows/publish.yaml

```
@@ -0,0 +1,41 @@
+name: Publish
+
+on:
+  workflow_call:
+
+jobs:
+
+  publish:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Log in to the Container registry
+        uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
```

This commit hash is associated with `V1.10.0`.
I think we could try the latest version of the action here.

```suggestion
        uses: docker/login-action@v2.2.0
```